src/main/java/games/strategy/engine/framework/ui/GameChooser.java

```diff
@@ -58,7 +59,7 @@ private void createComponents() {
     notesPanel = new JEditorPane();
     notesPanel.setEditable(false);
     notesPanel.setContentType("text/html");
-    notesPanel.setBackground(new JLabel().getBackground());
```

Comment:
I couldn't notice any difference with this.
I believe the "new" substance framwork is pretty consistent with its coloring.